src/main/cpp/t06_sum.cpp

```diff
@@ -21,8 +21,19 @@
 #include <cmath>
 
 using namespace std;
+int sum(int a)
+{
+	int sm = 0;
+	if (a == 0) { return sm; }
+	else
+	{		
+		int b = 0;
+		cin >> b;
+		sm = sm + b + sum(b);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'sm' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4519357)